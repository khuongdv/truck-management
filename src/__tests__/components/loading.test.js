import React from "react";
import { create } from "react-test-renderer";
import Loading from "../../components/loading/index";
describe("The Loading component", () => {
  test("it matches the snapshot", () => {
    const component = create(<Loading />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
