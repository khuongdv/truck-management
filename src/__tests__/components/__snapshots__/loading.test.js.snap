// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Loading component it matches the snapshot 1`] = `
Array [
  <div
    className="preloader-mask"
  >
    <div
      className="preloader"
    >
      <div
        className="spin base_clr_brd"
      >
        <div
          className="clip left"
        >
          <div
            className="circle"
            style={
              Object {
                "color": "#38ba72",
              }
            }
          />
        </div>
        <div
          className="gap"
        >
          <div
            className="circle"
          />
        </div>
      </div>
    </div>
  </div>,
  <div
    className="preloader-mask-container"
  />,
]
`;
